deny.toml

```diff
@@ -6,11 +6,7 @@
 # More documentation for the advisories section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html
 [advisories]
-vulnerability = "deny"
-unsound = "deny"
-unmaintained = "deny"
-yanked = "deny"
-notice = "warn"
+version = 2
```

Comment:
Are all the deletions in this file now handled by version 2?

Reply:
Yes; see the https://github.com/EmbarkStudios/cargo-deny/pull/611 linked above